metals/src/main/scala/scala/meta/internal/builds/GradleBuildTool.scala

```diff
@@ -21,9 +21,8 @@ case class GradleBuildTool(userConfig: () => UserConfiguration)
       ""
     }
   }
-  private val versionToUse = userConfig().bloopVersion
 
-  private val initScript =
+  private def initScript(versionToUse: String) =
```

Comment:
Since we are using string interpolation here it will be safer to check the type.